installers/dhall/launcher.dhall

```diff
@@ -1,26 +1,26 @@
 \(cluster : ./cluster.type) ->
 \(os      : ./os.type)      ->
 { configuration  =
-    { filePath     = "configuration.yaml"
-    , key          = cluster.key
+    { filePath     = os.configurationYaml
+    , key          = "${cluster.keyPrefix}_${os.name}"
     , systemStart  = [] : Optional Integer
     , seed         = [] : Optional Integer
     }
-, nodeLogConfig  = "log-config-prod.yaml"
 , nodeTimeoutSec = 30
 , reportServer   = cluster.reportServer
 , walletArgs     = [] : List Text
 , nodeArgs =
-    [ "--no-ntp"
-    , "--tlsca",               "tls/ca/ca.crt"
+    [ "--tlsca",               "tls/ca/ca.crt"
     , "--tlscert",             "tls/server/server.crt"
     , "--tlskey",              "tls/server/server.key"
-    , "--topology",            "wallet-topology.yaml"
     , "--update-server",       cluster.updateServer
     , "--keyfile",             os.nodeArgs.keyfile
     , "--logs-prefix",         os.nodeArgs.logsPrefix
+    , "--topology",            os.nodeArgs.topology
     , "--wallet-db-path",      os.nodeArgs.walletDBPath
     , "--update-latest-path",  os.nodeArgs.updateLatestPath
+    , "--wallet-address",      "127.0.0.1:8090"
+    -- XXX: this is a workaround for Linux
```

Comment:
..meaning to say, this is not strictly needed on non-Linux.

cc @cleverca22.

Reply:
what did it fail with when the option is omitted?

Reply:
Don't remember, just failed to connect to the node in some way, IIRC..

It came from your originally Nix-generated code, actually..